### ISSUE TITLE
Conversation/User details page - Blank header and Empty User details page displayed on accounts with a big amount of activity

### ISSUE THREAD

**Issue by isagoico** [Expensify/App#7664]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Log in with applausetester+0901abb@applause.expensifail.com / Feya86Katya 
2. Navigate to Concierge conversation
3. Click on the header to open the user details page

## Expected Result:
Header should display the first and last name (in this case only "Concierge") and user detail page should display all the user information. 

## Actual Result:
Header and user details page is blank. 

IMPORTANT NOTE: this is not reproducible in all accounts. Looks like only the ones that have a lot of chats and history are affected by this.

## Workaround:
Refreshing the page _sometimes_ fixes the issue.

## Platform:

Where is this issue occurring?

- Web

**Version Number:** v1.1.38-1 

**Reproducible in staging?:** Yes 
**Reproducible in production?:** Yes 

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

![image](https://user-images.githubusercontent.com/44479856/153300791-9f903e50-ff4e-4ca1-8df5-dd9b8556fd98.png)

https://user-images.githubusercontent.com/44479856/153301769-5dec969b-b6ab-4253-b65f-7929b88f6ae5.mp4

**Expensify/Expensify Issue URL:**

**Issue reported by:** @marcaaron and @thesahindia 
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1644435270721359

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @puneetlath (`Engineering`), see https://stackoverflow.com/c/expensify/questions/4319 for more details.

**Comment by isagoico** [comment]
Might be related to this bug https://github.com/Expensify/App/issues/7579. A lot of crazy behaviors have been reported and they seem to be affecting accounts with a lot of conversations and history only. 

I was unable to reproduce this on any of my test accounts (they don't have the same activity and conversations as the one mentioned in the OC)

**Comment by mallenexpensify** [comment]
I had this happen to me yesterday, at least the header part.  How to reproduce... no clue. 
I think @thesahindia is eligible for the reporting bonus since they reported it first [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1642707696241100), even though @thesahindia nor QA was able to reproduce.   The one caveat being... at @isagoico mentioned above, this could be related to another issue, possibly #7579. If so, a fix to that would also fix this without specifically referencing this issues.   

tl:dr... let's wait and see how this plays out before deciding if @thesahindia is due a bonus for reporting (apologies for alll the words)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @adelekennedy (`External`), see https://stackoverflow.com/c/expensify/questions/8582 for more details.

**Comment by adelekennedy** [comment]
Keeping my eye on #[7579](https://github.com/Expensify/App/issues/7579)!

**Comment by adelekennedy** [comment]
not overdue - still waiting on the other issue to resolve

**Comment by adelekennedy** [comment]
Not overdue - the other issue is still being resolved

**Comment by adelekennedy** [comment]
It looks like this may be [resolved](https://github.com/Expensify/App/issues/7579#issuecomment-1069405405) will check again tomorrow
